dataland-frontend/src/components/forms/parts/elements/derived/ActivityTree.ts

```diff
@@ -0,0 +1,970 @@
+import { type TreeNode } from "primevue/tree";
+
+export const activityTree: Array<TreeNode> = [
+  {
+    name: "Forestry",
+    key: "1",
+    sector: "Forestry",
+    type: "header",
+    children: [
+      {
+        name: "Afforestation",
+        value: "Afforestation",
+        key: "1.1",
+        sector: "Forestry",
+        type: "child",
+        nace_codes: "A2",
```

Comment:
Why use snake_case here when we are generally using camelCase?

Reply:
We take this data from:
https://ec.europa.eu/sustainable-finance-taxonomy/assets/taxonomy.json
and there is "nace_codes" so we may the same.

Reply:
How does the data from the json get into our code? I would expect it to go through a parser in the dev tools. If that is the case we can easily change the name of the variable and we should since this looks rather alien.

Reply:
I think it was created manually.

Reply:
changed to naceCodes